config

```diff
@@ -125,6 +125,9 @@
     # Fetch PR from GitHub by number/id
     fetchpr = "!sh -c 'git fetch origin pull/$0/head:pr/$0'"
 
+    # View different commits between branches
+    diffbranch = log --pretty=format:'%C(yellow)%h %Cred%ad %Cblue%an%Cgreen%d %Creset%s' --abbrev-commit --date=iso --graph
```

Comment:
How is it supposed to work?
From what I see it is not different than "hist" alias.
It does not show the difference between branches, just adds information about branches to commits, which is useful, but already present in both "llog" and "hist" aliases.
Or am I missing something